packages/contracts/contracts/Passport.sol

```diff
@@ -38,14 +42,16 @@ contract Passport is ERC721, Ownable {
     royaltyPcnt = _royaltyPcnt;
     royaltyRecipient = _owner;
     claimable = _claimable;
+    isPrivate = _isPrivate;
+    _setupRole(DEFAULT_ADMIN_ROLE, owner_);
```

Comment:
should we add the owner to the list of minters by default as part of constructor?
I wasn't sure if we can make that assumption.

Reply:
I'd leave off for now (less gas) and see what product feedback says